### PR TITLE
add trackBin to handleTransferBatch

### PR DESCRIPTION
**Pull request by cryptomeemaw**
- this pr addresses the bin totalSupply not being updated in `handleTransferBatch()`

### CODE REVIEW

File: src/lbPair.ts [hunk 1]
```diff
@@ -1037,6 +1037,34 @@ export function handleTransferBatch(event: TransferBatch): void {
       event.params.amounts[i],
       event.block
     );
+
+    // mint: increase bin totalSupply
+    if (ADDRESS_ZERO === event.params.from) {
```
Comment (gaepsuni) [inline src/lbPair.ts:1042]: `ADDRESS_ZERO.equals(event.params.from)` or `ADDRESS_ZERO == event.params.from`

File: src/lbPair.ts [hunk 2]
```diff
@@ -1037,6 +1037,34 @@ export function handleTransferBatch(event: TransferBatch): void {
...
+          BIG_DECIMAL_ZERO,
+          BIG_DECIMAL_ZERO,
+          BIG_DECIMAL_ZERO,
+          event.params.amounts[i], // minted
+          BIG_INT_ZERO
+      );
+    }
+
+    // burn: decrease bin totalSupply
+    if (ADDRESS_ZERO.toHexString() === event.params.to.toHexString()) {
```
Comment (gaepsuni) [inline src/lbPair.ts:1056]: `ADDRESS_ZERO.equals(event.params.to)`

**Review by gaepsuni — COMMENTED** [review]
I had a look at the docs and it looks like address equality is done either with the `==` operator or `.equals()`. We also need to change this in `handleTransferSingle`

File: src/lbPair.ts (hunk 2 above)
Comment (cryptomeemaw) [reply src/lbPair.ts:1056]: ty sir

**Review by gaepsuni — APPROVED** [review]

**Pull request merged**